addons/xm157/stringtable.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project name="ACE">
-    <Package name="XM157">
-    </Package>
-</Project>
+    <Package name="XM157"/>
```

Comment:
1. Is this valid xml? If not, I'll report it to Brett.
2. Since this has no translations, I was thinking of yeeting it. Thoughts?

Reply:
Yeah it's valid XML